bundles/org.eclipse.vorto.codegen.ui/src/org/eclipse/vorto/codegen/ui/handler/CodeGeneratorTaskExecutor.java

```diff
@@ -48,17 +50,19 @@ protected IStatus run(IProgressMonitor monitor) {
 				monitor.beginTask(
 						"Invoking generator [" + codeGenerator.getServiceKey() + "]",
 						1);
+				IGenerationResult generatedResult = null;
 				try {
-					IGenerationResult generatedResult = ((IVortoCodeGenerator) codeGenerator).generate(informationModel,invocationContext);
-					CodeGenerationHelper.createEclipseProject(ModelIdFactory.newInstance(informationModel), codeGenerator.getServiceKey(), generatedResult);
-					monitor.worked(1);
-					if (monitor.isCanceled()) {
-						monitor.done();
-						return Status.CANCEL_STATUS;
-					}
-
+					generatedResult = ((IVortoCodeGenerator) codeGenerator).generate(informationModel,invocationContext);
 				} catch (Exception e) {
-					e.printStackTrace();
+					GenerationResultZip output = new GenerationResultZip(informationModel,codeGenerator.getServiceKey());
```

Comment:
The Eclipse Client Toolset should not have an error.log. Instead it would be ideal, if the message is outputted to the Eclipse Console using MessageDisplay.error()